doc/api/tls.md

```diff
@@ -807,7 +807,7 @@ negotiation.
 Instances of `tls.TLSSocket` implement the duplex [Stream][] interface.
 
 Methods that return TLS connection metadata (e.g.
```

Comment:
Totally optional, but if you want to get rid of this `e.g.` too, that would be 👍 by me (and the [Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)).

```suggestion
Methods that return TLS connection metadata (such as
```